modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/ScalaClientCodegen.java

```diff
@@ -75,8 +80,8 @@ public ScalaClientCodegen() {
         importMapping.remove("Set");
         importMapping.remove("Map");
 
-        importMapping.put("DateTime", "org.joda.time.DateTime");
```

Comment:
@clasnake there's a PR by @epaul to add back the missing dependencies for joda time: https://github.com/swagger-api/swagger-codegen/pull/5092

If I remember correctly, we encountered issue with java.util.Date in the Java API client and therefore @cbornet introduced other datetime libraries (e.g. joda) to address the issue.